### PR TITLE
EasyConfig for the X11 bundle using the foss-2015a toolchain

### PR DESCRIPTION
**Pull request by valtandor**

### CODE REVIEW

**Comment by boegel** [comment]
@valtandor missing dep:

```
EasyBuildError: 'Irresolvable dependencies encountered: pkg-config/0.29.1-foss-2015a'
```

File: easybuild/easyconfigs/f/fontconfig/fontconfig-2.12.1-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+application access."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+source_urls = ['http://www.freedesktop.org/software/fontconfig/release/']
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('expat', '2.1.0'),      # Selected an older version for consistency
+    ('freetype', '2.6.2'),   # Selected an older version for consistency
```
Comment (boegel) [inline easybuild/easyconfigs/f/fontconfig/fontconfig-2.12.1-foss-2015a.eb]: @valtandor OK to drop the comments imho, it makes sense to use older dep versions for older toolchains

File: easybuild/easyconfigs/x/X11/X11-20160819-foss-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    'http://xkbcommon.org/download/',
+    'http://cgit.freedesktop.org/xorg/util/macros/snapshot',
+]
+
+builddependencies = [
+    ('Autotools', '20150215'),
+    ('pkg-config', '0.29.1'),
+]
+dependencies = [
+    ('freetype', '2.6.2'),    # Slightly older version than in foss-2016b for consistency
```
Comment (boegel) [inline easybuild/easyconfigs/x/X11/X11-20160819-foss-2015a.eb]: same here, ok to drop the comment imho

**Comment by valtandor** [comment]
@boegel this should be better.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 8 out of 23 (3 easyconfigs in this PR)
node2524.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/79cb7a046623f6ad487b9a0add1f6fa6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 3 (3 easyconfigs in this PR)
node2026.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f7e88916d77ce91e02a6a659ff0de063 for a full test report.

**Comment by boegel** [comment]
@valtandor hmm, please ignore the failing test reports for now, it seems like my test setup is kind of screwed up... I'll look into it.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2085.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/89a1bf40b8b57776d62033c33b76cc8c for a full test report.

**Comment by boegel** [comment]
problems with previous test reports were caused by testing #2834 at the same time while a whole bunch of dependencies with `foss/2015a` were missing

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (3 easyconfigs in this PR)
node2421.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7dfe05a11a81d916c8dda869b61d65f5 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @valtandor!

**Pull request merged**